app/forms/support_interface/conditions_form.rb

```diff
@@ -0,0 +1,78 @@
+module SupportInterface
+  class ConditionsForm
+    class OfferConditionField
+      include ActiveModel::Model
+
+      attr_accessor :id, :text
+
+      validates :text, length: { maximum: 255, too_long: ->(c, _) { "Condition #{c.id + 1} must be %{count} characters or fewer" } }
+    end
+
+    include ActiveModel::Model
+
+    attr_accessor :application_choice, :standard_conditions, :further_conditions, :audit_comment
+
+    NUMBER_OF_FURTHER_CONDITIONS = 4
```

Comment:
Is there any relationship between `NUMBER_OF_FURTHER_CONDTIONS` and `MAX_CONDITIONS_COUNT`? https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/services/make_an_offer.rb#L3

Reply:
Maybe there should be, I'm not sure why `MAX_CONDITIONS_COUNT` is as many as 20. This has changed recently, might need to rethink this a bit.

Reply:
Support users can now add up to 20 conditions (18 'further' conditions) though they can only add one new one at a time. We should revisit this once the JavaScript enhanced version is done in the provider interface.